wakuv2/waku.go

```diff
@@ -260,48 +264,86 @@ func New(nodeKey string, cfg *Config, logger *zap.Logger, appdb *sql.DB) (*Waku,
 	return waku, nil
 }
 
-func (w *Waku) addPeers(addresses []string, protocol libp2pproto.ID) {
+func (w *Waku) addPeers(addresses []string, protocol libp2pproto.ID, fnForEachPeer func(ma multiaddr.Multiaddr, protocol libp2pproto.ID)) {
```

Comment:
nit: create a type for the function such that you don't have to repeat: func(ma multiaddr.Multiaddr, protocol libp2pproto.ID) 3 times